src/Request/RequestFactory.php

```diff
@@ -78,10 +110,19 @@ private static function configureFactory(string $type): void
     }
 
     /**
+     * @deprecated since sonata-project/page-bundle 4.6.2, to be removed in 5.0.
      * @return class-string<Request>
      */
     private static function getClass(string $type): string
     {
+        @trigger_error(sprintf(
```

Comment:
The method is private so you don't need to deprecate it, it will naturally be remove.

Reply:
hmmm true 👀